.eslintrc.cjs

```diff
@@ -17,4 +17,7 @@ module.exports = {
     browser: true,
     node: true,
   },
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
I don't know if this is controversial, but I absolutely hate this option - if I'm adding an `any`, I'm already suppressing a typechecker warning. Then having to suppress the warning about the suppression feels strange.

Reply:
Yeah, I can live with that. It is annoying, and it's only ever been a false positive for me.

Reply:
Great, I'll have a separate PR to clean up the existing suppression comments for this rule.